sessionctx/variable/sysvar.go

```diff
@@ -708,10 +708,10 @@ var defaultSysVars = []*SysVar{
 		return nil
 	}},
 	/* The following variable is defined as session scope but is actually server scope. */
```

Comment:
Could we remove this comment?
Does this comment contain only line711 or several lines？ Do we need to update all related lines?
Besides, this PR do we need to do update these `session scope` variables or only add instance scope? Please add this info to the decription.

Reply:
Yes, we can remove this comment (previously the variables were organized, but this out of date).

Yes - there are several other system variables which need to be changed to instance scope. In an earlier review comment I suggested that we just update one variable first, since this file often has lots of merge conflicts. You can see the revert here: https://github.com/pingcap/tidb/pull/32888/commits/4538447bd726b8ee0d94c9a0d2beaa1cd901e0e4 -- we will need to address this in a followup PR.

Reply:
I will make another PR to re-organize the order of system variables by their scopes.

Reply:
Got it. Maybe we can add this info to the description.